lib/eb_deployer/resource_stacks.rb

```diff
@@ -1,15 +1,19 @@
 module EbDeployer
   class ResourceStacks
-    def initialize(resources, cf_driver, skip_provision=false)
+    def initialize(resources, cf_driver, skip_provision=false, tags)
       @resources = resources
       @cf_driver = cf_driver
       @skip_provision = skip_provision
+      @tags = {}
```

Comment:
Can be idiomatically simplified to `@tags = (tags || {}).map { |k, v| { key: k, value: v } }`